MC-2655 - update map config limitations (#298)
hazelcast/management-center-docs: docs/modules/data-structures/pages/map.adoc

You can change any attribute and click the **Update** button to save
your changes.

> NOTE: Map config is updated only on the members MC is aware of at the time of the update. When a new member joins the cluster, the updated map configuration is not propagated to it.

Reviewer: ```suggestion
NOTE: The Map configuration is updated only on the members that Management Center is aware of at the time of the update. When a new member joins the cluster, the updated map configuration is not propagated to it.
```

> Also if a member has persistence enabled, the updated map config is not persisted to disk and the updates are lost upon the next restart.

Reviewer: ```suggestion
If a member has persistence enabled, the updated map configuration is not persisted to disk and the updates are lost on the next restart.
```
